pkg/domain/ecosystem/EcosystemState.go

```diff
@@ -2,18 +2,13 @@ package ecosystem
 
 import (
 	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain/common"
-	"golang.org/x/exp/maps"
+	"github.com/cloudogu/k8s-registry-lib/config"
 )
 
 type EcosystemState struct {
-	InstalledDogus               map[common.SimpleDoguName]*DoguInstallation
-	InstalledComponents          map[common.SimpleComponentName]*ComponentInstallation
-	GlobalConfig                 map[common.GlobalConfigKey]*GlobalConfigEntry
-	DoguConfig                   map[common.DoguConfigKey]*DoguConfigEntry
-	EncryptedDoguConfig          map[common.SensitiveDoguConfigKey]*SensitiveDoguConfigEntry
-	DecryptedSensitiveDoguConfig map[common.SensitiveDoguConfigKey]common.SensitiveDoguConfigValue
-}
-
-func (state EcosystemState) GetInstalledDoguNames() []common.SimpleDoguName {
```

Comment:
So this is not needed anymore?

Reply:
Correct, we can also use the functions for normal dogu config for sensitive config as well. This will be refactored further this week.